tests/python/gaia-ui-tests/gaiatest/tests/functional/settings/manifest.ini

```diff
@@ -80,8 +80,7 @@ online = true
 skip-if = device == "desktop" && os == "linux" && debug
 
 [test_settings_sim_manager.py]
-disabled = Disabled because run the device with 1 SIM only
-run-if = device == "sp7710gaplus_gonk" || device == "flame"
+dsds = true
```

Comment:
Currently, DSDS implies that you have 2 SIM cards and we only need that in this test. Carrier is not necessary here then.